### PR TITLE
Migrate the CI setup to Github actions

### PR DESCRIPTION
**Pull request by stof**

### CODE REVIEW

**Comment by stof** [comment]
See https://github.com/stof/sass-spec/actions/runs/466185101 for a test run in my own fork (PRs opened from a work won't run a workflow they introduce, due to some security restrictions).

I made 2 changes compared to the Travis setup:
- I added a job running the unit tests of the TS runner (I'm quite sure not having that job was a mistake when introducing the TS runner rather than an explicit choice)
- The skipping of jobs does not support passing a comma-separated list of implementations in the PR description anymore after the `skip` keyword (the github expression syntax for disabling jobs in the clean way does not support regexps). To skip multiple implementations for the same PR, the `skip` keyword needs to be repeated for each of them instead (I considered this trade-off OK since most PRs are only skipping one implementation anyway)

**Comment by stof** [comment]
A future improvement here would be caching the compilation of sassc, given that this is what takes most of the time in this job and libsass does not change often (especially now that it is deprecated).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      # Feature branches beginning with "feature."
+      - feature.*
+  pull_request:
```
Comment (nex3) [inline .github/workflows/ci.yml:9]:
```suggestion
  pull_request: {branches: ['**']}
```

Otherwise I don't think this will actually run.

**Review by nex3 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (stof) [reply .github/workflows/ci.yml:9]: it will run. When you don't define `branches` or `tags`, it runs on all references

Comment (nex3) [reply .github/workflows/ci.yml:9]: That wasn't my experience with https://github.com/sass/sass/pull/2992... it didn't run until I pushed https://github.com/sass/sass/pull/2992/commits/3386ec27dcf3cb726ae3984dfdfdb70d8a75b2a0.

Comment (stof) [reply .github/workflows/ci.yml:9]: then the github doc would be wrong

Comment (stof) [reply .github/workflows/ci.yml:9]: also, I have other repos where this works.

Comment (nex3) [reply .github/workflows/ci.yml:9]: Looks like you're right, https://github.com/sass/sass/pull/2998 is running actions now. No idea why my previous attempt didn't work.

**Comment by nex3** [comment]
Forked to #1612 to get the CI running.

**Pull request closed without merging**